code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -119,6 +119,15 @@
 	e.start()
 	holder.clear_reagents()
 
+/datum/reagent/anfo
+	name = "Ammonium Nitrate"
+	description = "High explosive fertilizer"
```

Comment:
you should actually make it a fertilizer, see saltpetre and the nutriments in `modules/reagents/chemistry/reagents/other_reagents.dm`

Reply:
ah
forgot about that